x/supplier/keeper/params_test.go

```diff
@@ -60,3 +71,44 @@ func TestParams_ValidateMinStake(t *testing.T) {
 		})
 	}
 }
+
+func TestParams_ValidateStakingFee(t *testing.T) {
+	tests := []struct {
+		desc        string
+		minStake    any
+		expectedErr error
+	}{
+		{
+			desc:        "invalid type",
+			minStake:    "420",
+			expectedErr: suppliertypes.ErrSupplierParamInvalid.Wrapf("invalid parameter type: string"),
+		},
+		{
+			desc: "SupplierStakingFee less than zero",
+			minStake: &cosmostypes.Coin{
+				Denom:  volatile.DenomuPOKT,
+				Amount: math.NewInt(-1),
+			},
+			expectedErr: suppliertypes.ErrSupplierParamInvalid.Wrapf(
+				"staking_fee amount must be positive: got -1%s",
+				volatile.DenomuPOKT,
+			),
+		},
+		{
+			desc:     "valid SupplierStakingFee",
+			minStake: &suppliertypes.DefaultStakingFee,
+		},
```

Comment:
If we're intentionally allowing a `0` staking fee, we should have a test for it.